perf/benchmarkapps/BenchmarkServer/hosting.json

```diff
@@ -0,0 +1,6 @@
+{
+  "urls": "",
+  "server.urls": "",
+  "UseTls": "false",
```

Comment:
If we want to disable TLS, maybe we should just not specify a certificate instead?

Reply:
This is easily settable using the benchmark framework.